prestodb/centos6-oj8/Dockerfile

```diff
@@ -46,6 +46,9 @@ RUN \
            $JDK_PATH/jre/lib/amd64/libjavafx*.so \
            $JDK_PATH/jre/lib/amd64/libjfx*.so && \
     
+    # make java fixed (so further layers don't accidentally override it)
+    update-alternatives --set java $JDK_PATH/jre/bin/java && \
```

Comment:
how does it make it fixed? About what accidents you are talking about?

I thought rpm sets it for you already, isn't?

Reply:
rpm sets this only in a sense, that subsequent rpm (e.g. with openjdk) will override it.

@ilfrin did you verify that installing openjdk won't change alternative if has been explicitly set?

Reply:
I didn't check that, but I did confirm that previously it was auto mode, and that the openjdk had a higher number, and was picked up automatically, this should prevent that

Reply:
@findepi I've run `yum install java-1.8.0-openjdk.x86_64` and the update-alternatives setting survives that, it's still `manual`, and it's still pointing to the Oracle version, despite a new option available with a higher priority. It would get chosen if it was in `auto` mode